src/Microsoft.Dnx.Runtime.Sources/Impl/EntryPointExecutor.cs

```diff
@@ -90,7 +90,8 @@ private static bool TryGetEntryPoint(Assembly assembly, IServiceProvider service
 
                 if (programTypeInfo == null)
                 {
-                    System.Console.WriteLine("'{0}' does not contain a static 'Main' method suitable for an entry point", name);
+                                        
+                    System.Console.WriteLine("'{0}' does not contain a type 'Program' for an entry point", name);
```

Comment:
Slight reword suggestion: '{0}' does not contain a 'Program' type suitable for an entry point